Content.Server/Remotes/DoorRemoteSystem.cs

```diff
@@ -0,0 +1,92 @@
+            SubscribeLocalEvent<DoorRemoteComponent, AfterInteractEvent>(OnAfterInteract);
+        }
+
+        public void OnInHandActivation(EntityUid user, DoorRemoteComponent component, UseInHandEvent args)
+        {
+            switch (component.Mode)
+            {
+                case DoorRemoteComponent.OperatingMode.OpenClose:
+                    component.Mode = DoorRemoteComponent.OperatingMode.ToggleBolts;
+                    _popupSystem.PopupEntity(Loc.GetString("door-remote-switch-state-toggle-bolts"), args.User, Filter.Entities(args.User));
+                    break;
+                case DoorRemoteComponent.OperatingMode.ToggleBolts:
+                    component.Mode = DoorRemoteComponent.OperatingMode.OpenClose; // TODO: Swítch to ToggleEmergencyAcces when EA is implemented
+                    _popupSystem.PopupEntity(Loc.GetString("door-remote-switch-state-open-close"), args.User, Filter.Entities(args.User)); // TODO: See the above comment
+                    break;
+            /*
+                case DoorRemoteComponent.OperatingMode.ToggleEmergencyAccess:
+                    component.Mode = DoorRemoteComponent.OperatingMode.OpenClose;
+                    _popupSystem.PopupEntity(Loc.GetString("door-remote-switch-state-open-close"), args.User, Filter.Entities(args.User));
+                    break;
+            */
+            }
+        }
+        private void OnAfterInteract(EntityUid uid, DoorRemoteComponent component, AfterInteractEvent args)
+        {
+            if (args.Handled
+                || args.Target == null
+                || !EntityManager.TryGetComponent<DoorComponent>(args.Target, out var doorComponent) // If it isn't a door we don't use it
+                || !EntityManager.HasComponent<AccessReaderComponent>(args.Target) // Remotes do not work on doors without access requirements
+                || !EntityManager.TryGetComponent<AirlockComponent>(args.Target, out var airlockComponent) // Remotes only work on airlocks
+                || !_interactionSystem.InRangeUnobstructed(args.User, doorComponent.Owner, -1f, CollisionGroup.Opaque))
+
+            {
+                return;
+            }
+
+            args.Handled = true;
+            
+            if (component.Mode == DoorRemoteComponent.OperatingMode.OpenClose)
+            {
+                _sharedDoorSystem.TryToggleDoor(doorComponent.Owner, user: args.Used);
+            }
+
+            if (component.Mode == DoorRemoteComponent.OperatingMode.ToggleBolts
+                && airlockComponent.IsPowered())
+            {
+                if (_doorSystem.HasAccess(args.Used))
+                {
+                    if(airlockComponent.IsBolted())
+                    {
+                        airlockComponent.SetBoltsWithAudio(false);
+                    }
+                    else
+                    {
+                        airlockComponent.SetBoltsWithAudio(true);
+                    }
```

Comment:
maybe just condense this to `airlockComponent.SetBoltsWithAudio(!airlockComponent.IsBolted());`